tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -528,22 +528,20 @@ public function test_payment_fields_renders_currency_selector_conditionally(
+			$this->assertNotFalse( $selector_position, 'Currency selector position should be detectable.' );
 			$this->assertNotFalse( $upe_element_position, 'Payment element should be present in output.' );
 			$this->assertLessThan(
 				$upe_element_position,
-				$currency_selector_position,
+				$selector_position,
 				'Currency selector should render before the payment element.'
 			);
 		} else {
-			$this->assertStringNotContainsString( $currency_selector_wrapper, $output );
+			$this->assertStringNotContainsString( $selector_div, $output );
 		}
```

PHPMD (2.15.0)

[error] The method test_payment_fields_renders_currency_selector_conditionally uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)